src/core/PointLight.ts

```diff
@@ -1,38 +1,19 @@
 import {PointLight as ThreePointLight} from 'three'
 import LightBase from './LightBase'
-import {props} from './props'
-import {mapPropTo} from './props'
 import {prop} from '../html/WithUpdate'
 
 export default class PointLight extends LightBase {
     static defaultElementName = 'i-point-light'
 
-    @prop(mapPropTo({...props.number, default: 0}, (self: any) => self.three))
-    distance!: number
-
-    @prop(mapPropTo({...props.number, default: 1}, (self: any) => self.three))
-    decay!: number
-
-    @prop(mapPropTo({...props.boolean, default: true}, (self: any) => self.three))
-    castShadow!: boolean
-
-    @prop({...props.number, default: 512})
-    shadowMapWidth!: number
-
-    @prop({...props.number, default: 512})
-    shadowMapHeight!: number
-
-    @prop({...props.number, default: 3})
-    shadowRadius!: number
-
-    @prop({...props.number, default: 0})
-    shadowBias!: number
-
-    @prop({...props.number, default: 1})
-    shadowCameraNear!: number
-
-    @prop({...props.number, default: 2000})
-    shadowCameraFar!: number
+    @prop(Number) distance = 0
+    @prop(Number) decay = 1
+    @prop(Boolean) castShadow = true
+    @prop(Number) shadowMapWidth = 512
+    @prop(Number) shadowMapHeight = 512
+    @prop(Number) shadowRadius = 3
+    @prop(Number) shadowBias = 0
+    @prop(Number) shadowCameraNear = 1
+    @prop(Number) shadowCameraFar = 2000
```

Comment:
@corruptedzulu getting cleaner!

Reply:
However, I'm about to add documentation with jsdocs, so things like

```ts
@prop(Number) distance = 0
@prop(Number) decay = 1
```

may become things like

```ts
/**
 * The distance at which light strength is zero. If set to zero ...
 */
@prop(Number) distance = 0

/**
 * When distance is not zero, decay specifies the rate at which light intensity diminishes. ...
 */
@prop(Number) decay = 1
```